cmd/snap/cmd_snap_op.go

```diff
@@ -369,9 +369,12 @@ func showDone(cli *client.Client, names []string, op string, opts *client.SnapOp
 				msg = fmt.Sprintf(i18n.G("%q left the cohort"), snap.Name)
 			}
 			fmt.Fprintln(Stdout, msg)
+			// go to next snap, skipping the temporary forward logic below for "switch" op.
```

Comment:
I think we should expand the existing unit test a wee bit, something like:
```
diff --git a/cmd/snap/cmd_snap_op_test.go b/cmd/snap/cmd_snap_op_test.go
index 54212caad5..d95eab704e 100644
--- a/cmd/snap/cmd_snap_op_test.go
+++ b/cmd/snap/cmd_snap_op_test.go
@@ -1890,7 +1890,9 @@ func (s *SnapOpSuite) TestSwitchHappy(c *check.C) {
        rest, err := snap.Parser(snap.Client()).ParseArgs([]string{"switch", "--beta", "foo"})
        c.Assert(err, check.IsNil)
        c.Assert(rest, check.DeepEquals, []string{})
-       c.Check(s.Stdout(), check.Matches, `(?sm).*"foo" switched to the "beta" channel`)
+       c.Check(s.Stdout(), check.Equals, `"foo" switched to the "beta" channel
+
+`)
        c.Check(s.Stderr(), check.Equals, "")
        // ensure that the fake server api was actually hit
        c.Check(s.srv.n, check.Equals, s.srv.total)
```

Reply:
I'm not sure I see why your suggestion is better.
In any case, ideally checks to Stdout would use the wrapping-aware checkers from testutil, but I'm not sure it's a win for this PR.

Reply:
This one is using "equals" vs "matches", I think this is important because the old test did not catch the regression because the contains was still fine but an equals would have caught this.